src/main/java/fi/helsinki/cs/tmc/core/commands/VerifyCredentials.java

```diff
@@ -49,7 +50,9 @@ public Boolean call() throws TmcCoreException, IOException {
 
         int response =
                 communicator
-                        .makeGetRequest(settings.getServerAddress() + TMC_SERVER_ROUTE, auth)
+                        .makeGetRequest(
+								URI.create(settings.getServerAddress() + TMC_SERVER_ROUTE),
+								auth)
```

Comment:
This seems like too much indentation.

I also noticed you have used tabs here instead of 4x spaces. You should be able to change your IDE to automatically "use spaces instead of tabs" and configure the correct amount of spaces per tab.